README.md

```diff
@@ -1074,7 +1074,51 @@ RUN mkdir -p -m755 /home/seluser/.pki/nssdb \ #create nssdb folder
     && pk12util -d sql:/home/seluser/.pki/nssdb -i client_cert.p12 -W password_of_clent_cert # client certificate install
 ```
 This way the certificates will be installed and the node will start automatically as before.
+___
+
+## Alternative method: Add certificates to existing Selenium based images for browsers
+
+As an alternative, you can add youe certificate files to existing Selenium images. This practical example
+assumes you have a known image to use as a build image and have a way to publish new images to your local
+docker registry.
+
+This also assumes you are using a redhat based image as *build* _with_ your internal CA already added.
+(default location for rocky linux is /etc/pki/ca-trust/source/anchors/YOUR_CA.pem)
```

Comment:
The issue I see is that users will ask how to build this image. Does it need to be RedHat? Could it be any Linux based image?

Reply:
What I mean is... would it be possible to add the instructions to build that image?

Reply:
It can be any image, the actual instructions may vary.
i'll clarify this

Reply:
An example is always more helpful